mitodl/mitxonline
# Added minimal learner dashboard

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #5

#### What's this PR do?
Adds a minimal learner dashboard so a user can see their enrolled courses

#### How should this be manually tested?
- Add some `CourseRunEnrollment` records for one of your users
- Add a course page for the corresponding course in Wagtail
- Check the `/dashboard/` page, ensuring that the title links to the edX course if `courseware_url` is set, it matches the designs in all screen widths, and the `start_date` values are displayed appropriately

#### Any background context you want to provide?
Designs: https://projects.invisionapp.com/share/KH11D3ZN53GC#/screens/455535008

#### Screenshots (if appropriate)
![ss 2021-07-23 at 09 54 35 ](https://user-images.githubusercontent.com/14932219/126800512-366c54ea-9224-417d-ab20-cc9af2acd17e.png)
![ss 2021-07-23 at 08 10 11 ](https://user-images.githubusercontent.com/14932219/126786652-551a7424-20a6-4766-8d87-8a48ff6763a3.png)
